app/src/Event/EventController.php

```diff
@@ -36,6 +36,7 @@ protected function defineRoutes(Slim $app)
         $app->get('/event', array($this, 'index'))->name("events-index");
         $app->get('/event/pending', array($this, 'pending'))->name("events-pending");
         $app->map('/event/submit', array($this, 'submit'))->via('GET', 'POST')->name('event-submit');
+        $app->map('/event/:friendly_name/import', array($this, 'eventImportCsv'))->via('GET', 'POST')->name("event-import-csv");
```

Comment:
Can we rename that endpoint to "importcsv" (or "import/scv")? That will allow us to add an XML-Import or any other kind of import later without having to change the API.

Alternatively the endpoint should check the fileformat and decide based on that what kind of import to use. But that'd IMO require the method to be renamed to "eventImport"

Reply:
I have no plans to support anything but CSV importing. If someone wanted to add XML importing in the future, I'd push back and not want to maintain 2 import formats.